src/main/java/com/example/greeting/controller/HelloController.java

```diff
@@ -0,0 +1,12 @@
+package com.example.greeting.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class HelloController {
+    @RequestMapping(value = "/hello")
```

Comment:
httpリクエストメソッドのGETを受け付けるということを明記しておくのが良いかと思います。
```suggestion
    @GetMapping("/hello")
```

Reply:
確かにGetMappingでした。
修正済みです。